integration/shareddata/bignumbers.go

```diff
@@ -0,0 +1,26 @@
+// Copyright 2021 FerretDB Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package shareddata
+
+// BigNumbersData contains big numbers of different types for tests.
+//
+// This shared data set is not frozen yet, but please add to it only if it is really shared.
+var BigNumbersData = &Values[string]{
+	data: map[string]any{
+		"int64-big":  int64(2 << 61),
+		"double-big": float64(2 << 60),
+		"int32-big":  int32(2 << 25),
```

Comment:
All int32 values can be represented by float64 values exactly, so that do not anyhting, I think? We probably can remove it

Reply:
Removed